src/js/components/homepageUpdate/hero/AnimatedHeading.jsx

```diff
@@ -0,0 +1,155 @@
+import React, { useEffect, useState } from "react";
+
+const AnimatedHeading = ({ paused }) => {
+    const leftWords = ['Explore', 'Search', 'Track', 'Download', 'Analyze'];
+    const rightWords = ['by industry', 'by agency', 'over time', 'to communities', 'by recipient'];
+    const [endWordTop, setEndWordTop] = useState();
+    // TODO with DEV-8677
+    // const getRandomInt = (max) => Math.floor(Math.random() * max);
+
+    const rotatingWords = {
+        left: {
+            index: null,
+            previousWord: null,
+            wordsArray: [...leftWords],
+            tempWordsArray: [...leftWords],
+            currentWord: null
+        },
+        right: {
+            index: null,
+            previousWord: null,
+            wordsArray: [...rightWords],
+            tempWordsArray: [...rightWords],
+            currentWord: null
+        }
+    };
+
+    // TODO with DEV-8677
+    // const pickWord = (position) => {
+    //     const { wordsArray } = rotatingWords[position];
+    //
+    //     let {
+    //         index, previousWord, tempWordsArray
+    //     } = rotatingWords[position];
+    //
+    //     if (previousWord) {
+    //         tempWordsArray.splice(tempWordsArray.indexOf(tempWordsArray[index]), 1);
+    //     }
+    //
+    //     if (tempWordsArray?.length === 0) {
+    //         tempWordsArray = [...wordsArray];
+    //     }
+    //
+    //     do {
+    //         index = getRandomInt(tempWordsArray.length);
+    //     } while (previousWord === tempWordsArray[index]);
+    //
+    //     previousWord = tempWordsArray[index];
+    //     rotatingWords[position].currentWord = previousWord;
+    //     return index;
+    // };
+
+    useEffect(() => {
+        const intro = document.querySelectorAll(".phrase__intro__item span");
+        intro.forEach((item, index) => {
+            intro[index].style.animationPlayState = paused ? "paused" : "running";
```

Comment:
if i'm understanding this correctly paused is going to be only true or false, is there a reason why you need them to be ternary functions? i'm not seeing any follow on logic for when it is "paused" or "running" 

but I might be missing something

Reply:
The animation-play-state accepts either "paused" or "running".  I have to toggle the play state to either paused or running based on the current state and if the user pressed "paused" or "play"

https://developer.mozilla.org/en-US/docs/Web/CSS/animation-play-state

Reply:
well that would be the thing i was missing!